EntAppProject/src/main/java/com/jobook/EntAppProject/Services/JobServiceStub.java

```diff
@@ -12,48 +12,46 @@
 
 @Component
 public class JobServiceStub implements IJobService {
-	
+
 	@Autowired
 	IUserService userService;
-	
-	private List<Job> jobs; 
-	
-	private void generateTestJobs()
+
+	private List<Job> generateTestJobs()
 	{
-		jobs = new ArrayList<Job>();
+		List<Job> jobs = new ArrayList<>();
 		Customer customer = (Customer) userService.getUserInfoById(0);
 		Employee employee = (Employee) userService.getUserInfoById(1);
 		jobs.add(new Job(0, customer.getId(), null, customer, null, "This is Test Job 1.", "Test Job 1", null, null, 100.0));
 		jobs.add(new Job(1, customer.getId(), employee.getId(), customer, employee, "This is Test Job 2.", "Test Job 2", null, null, 300.0));
 		jobs.add(new Job(2, null, employee.getId(), null, employee, "This is Test Job 3.", "Test Job 3", null, null, 500.0));
-		
+		return jobs;
 	}
-	
+
 	public List<Job> getUsersRelatedJobs(int userId) {
-		generateTestJobs();
-		List<Job> filteredJobs = new ArrayList<Job>();
-		
+		List<Job> jobs = generateTestJobs();
```

Comment:
Extending on my comment above, scope the variables to the method (like so), so you do not have to deal with shadowing issues.